netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
-        })
+
+        # Lookup for webos3
+        webos_entry = self.find_by_device_description(
+            {
+                "deviceType": "urn:schemas-upnp-org:device:MediaRenderer:1",
+                "manufacturer": "LG Electronics.",
+                "modelName": "LG TV"
+            }
+        )
+
+        if not len(webos_entry):
```

Comment:
The problem is exactly that it is either one or the other. That means that it is not deterministic. So let's just remove it and validate with the original author that his discovery still works.